srcpkgs/taisei/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'taisei'
 pkgname=taisei
-version=1.3.2
-revision=2
+version=1.4.1
+revision=1
 build_style=meson
-hostmakedepends="pkg-config python3-docutils"
+hostmakedepends="pkg-config python3-docutils python3-zstandard"
 makedepends="SDL2_mixer-devel freetype-devel libwebp-devel libzip-devel cglm
```

Comment:
Taisei does not need SDL2_mixer since v1.4. Also missing here are libpng and zlib.